Libraries/LibC/stdlib.hpp

```diff
@@ -50,4 +49,10 @@ static int bsearch(int elem, int arr[], int count, int start = 0)
     }
     return -1;
 }
+
+template<typename T>
+T abs(T num){
```

Comment:
Not a big issue by any means but the function should be formatted as such:

`T abs(T num)`
`{`

With a new line before the opening bracket.

Reply:
yeah but it is forcing all the types to be int. For example user may pass float?